electron/src/lib/openGraph.ts

```diff
@@ -146,13 +146,15 @@ const fetchOpenGraphData = async (url: string): Promise<OpenGraphResult> => {
   };
 
   const body = await axiosWithContentLimit(axiosConfig, CONTENT_SIZE_LIMIT);
-  const [head] = body.match(/<head>[\s\S]*?<\/head>/) || [''];
+  const matches = body.match(/.*property="og:[^"]+".*/gim) || [''];
```

Comment:
https://regex101.com/r/kwrYZD/1

Reply:
what about single quotes?
```suggestion
  const matches = body.match(/.*property=(["'])og:.+?\1.*/gim) || [''];
```